less/navbar.less

```diff
@@ -0,0 +1,12 @@
+
+// .jumbo-home {
+//     position: relative; 
+// }
+
+.nav {
+   position: fixed;
+   top: 0;
```

Comment:
with position fixed you don't have to set the top to 0, but its not hurting anything either.